homeassistant/helpers/entity_registry.py

```diff
@@ -152,30 +152,35 @@ def hidden(self) -> bool:
         """Return if entry is hidden."""
         return self.hidden_by is not None
 
+    @property
+    def dict_repr(self) -> dict[str, Any]:
```

Comment:
Wouldn't a more common name for such a function be: `as_dict`?

Reply:
Yes, but I don't want this function to accidentally be used by our JSON decoder.
Maybe we could call it `as_basic_dict` or `as_limited_dict`?